src/main/java/uk/gov/hmcts/reform/civil/handler/tasks/CaseEventTaskHandler.java

```diff
@@ -89,6 +89,7 @@ private String getSummary(String eventId, String state) {
                 case COUNTER_CLAIM:
                     return "RPA Reason: Defendant rejects and counter claims.";
                 case PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT:
+                case PENDING_CLAIM_ISSUED_UNREPRESENTED_UNREGISTERED_DEFENDANT:
```

Comment:
May need to refactor this in future to have its own custom wording. But fine for now!

Reply:
Yep, it's agreed that RPA messages are for a different story

Reply:
Can we leave a "to do" comment here @sherlynkhaw